src/app/core/header/header.component.html

```diff
@@ -1,27 +1,19 @@
-<header class="navbar navbar-light bg-light p-0">
-  <div class="container-fluid">
-    <a class="navbar-brand" href="#">
-      <img src="http://placehold.it/300x60?text=Video Portal" width="150" height="30" alt="Video Portal">
-    </a>
-    <ul class="navbar-nav ml-auto">
-      <li class="nav-item dropdown mr-3">
-        <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown">
-          <i class="fa fa-user"></i> {{currentUser.name}}
-        </a>
-        <div class="dropdown-menu">
-          <a href="#" class="dropdown-item">
-            <i class="fa fa-user-circle"></i> Profile
-          </a>
-          <a href="#" class="dropdown-item">
-            <i class="fa fa-gear"></i> Settings
-          </a>
-        </div>
+<mat-toolbar color="primary">
+  <div><a href="#">LOGO</a></div>
+  <div fxFlex fxLayout fxLayoutAlign="flex-end" fxHide.xs>
+    <ul fxLayout fxLayoutGap="10px" class="navigation-items">
+      <li *ngIf="!isAuth">
```

Comment:
Better to use `ng-container` with `*ngIf` + `else` combination instead duplications

Reply:
Done.